src/autolabel/configs/config.py

```diff
@@ -169,6 +170,10 @@ def model_params(self) -> Dict:
         """Returns a dict of configured settings for the model (e.g. hyperparameters)"""
         return self._model_config.get(self.MODEL_PARAMS_KEY, {})
 
+    def model_endpoint(self) -> str:
+        """Returns the endpoint to use for the model"""
+        return self._model_config.get(self.MODEL_ENDPOINT_KEY, None)
```

Comment:
@yadavsahil197 to ensure this code change works for RefuelLLMv2 large and small models, we'll have to change the elb endpoint (to reflect llm.refuel.ai/models/.../v2/generate) in their LLMEntity tables, correct?

Reply:
The base model configs already have the updated endpoints.
For finetuned models, we will need to set the elb endpoint to the correct values.

Reply:
Also, this doesn't change the behavior for realtime requests, as the inference server continues to use the earlier deployed model.